modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -20,16 +20,31 @@
 	<td>//div[@class='dropdown-menu show']//button[contains(text(),'Change Publication Status')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CONTINUE_WITHOUT_SAVING_BUTTON</td>
+	<td>//button[contains(text(),'Continue Without Saving')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MODAL_CREATE_BUTTON</td>
```

Comment:
This is already added in master with a different name, rebase to master to resolve any conflicts.